### PR TITLE
[WIP] Alt-Manager

### PR DESCRIPTION
**Pull request by HerXayah**
### Done

[ ] - Raw Idea of what the fuck i wanna do

### Working on

[ ] - Alt List
[ ] - Finishing TopBar
[ ] - Fixing Tooltips
[ ] - Implement Dropdowns

### Screenshots

**19.06.2023**
image

### CODE REVIEW

**Comment by be4dev** [comment]
> ### Done
> 
> [ ] - Raw Idea of what the fuck i wanna do
> ### Working on
> 
> [ ] - Alt List [ ] - Finishing TopBar [ ] - Fixing Tooltips [ ] - Implement Dropdowns
> ### Screenshots
> 
> **19.06.2023** image

well, i used to work a bit on proxy manager, so I have some steps done, here you go https://github.com/CCBlueX/LiquidBounce/commit/05ad8626f3ce4c1cb783c94c6116ed6b6b99c447
image
hope it really looks better and can help you

**Comment by HerXayah** [comment]
Inb4 converting it to svelte.

Also need to cleanup ur code 👌👍

**Comment by HerXayah** [comment]
TLDR:
Will work on it again when uni not yeeting my whole life 👌👍
abt 3 weeks

**Comment by 1zun4** [comment]
Replaced by https://github.com/CCBlueX/LiquidBounce/pull/1442
